src/pages/Authentication/Signup.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Navbar } from '../../components'
+// import './Home.css'
+
+export const Signup = () => {
+  return (
+    <main className="login_page">
+      <Navbar LoginOrSignup="Login" />
+
+      <section className="login_box content">
+        <form>
+          <div className="login_div">
+            <h2 className="createAccount">Signup</h2>
+            <label for="login_input">First Name</label>
+            <input
+              type="text"
+              className="login_input"
+              placeholder="Enter you first name"
+              required
+            />
```

Comment:
Whenever there is label and input together you can give id to input and htmlFor to labels their values being same, this helps in accessibility.